README.md

```diff
@@ -78,17 +77,13 @@ After you're finished please remove all the comments and instructions!
 ### Key Features <a name="key-features"></a>
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230422685-4b96f36f-421a-4eb4-9734-fa38f532bcad.png)

**[OPTIONAL]**

- [ ] It would be great if you add your Twitter account handle as well because a lot of recruiters connect with people on Twitter 🤔